modules/mod_translation/support/translation_scan.erl

```diff
@@ -72,14 +72,17 @@ merge_args([{Lang,Text}|Rest], Args) ->
 
 %% @doc Parse the template or Erlang module. Extract all translation tags.
 scan_file(<<".tpl">>, File) ->
-    scan_file(".tpl", File); 
+    scan_file(".tpl", File);
 scan_file(".tpl", File) ->
     case template_compiler:translations(File) of
-        {ok, Translations} -> normalize_line_info(Translations);
+        {ok, Translations} ->
+            %% Trim translation strings because template compiler includes spaces
+            Trimmed = [{z_string:trim(Label), Trans, FileInfo} || {Label, Trans, FileInfo} <- Translations],
```

Comment:
@mworrell This was needed because the message labels coming in from the template compiler have outside spaces, which resulted in duplicate messages. Should we update the compiler to do the trimming itself?